miner/envs/runner/_build-image.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -eux
```

Comment:
https://github.com/reef-technologies/handbook/blob/master/docs/Code_conventions.md#safe-shell-settings
```suggestion
#!/bin/bash
set -eux -o pipefail
```
use of `-x` is script-context dependent, but rest are "sane defaults" 

please apply to other places as well (FYI  if you have any non-bash scripts as well - `sh` doesn't support `-o pipefail`)

Reply:
fixed